src/org/zaproxy/zap/extension/ascan/ActiveScanAPI.java

```diff
@@ -537,6 +542,15 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
 					throw new ApiException(ApiException.Type.ILLEGAL_PARAMETER, PARAM_IDX, e);
 				}
 				break;
+			case ACTION_SKIP_SCANNER:
+				int pluginId = getParam(params, PARAM_SCANNER_ID, -1);
+				if (pluginId == -1) {
+					throw new ApiException(ApiException.Type.ILLEGAL_PARAMETER, PARAM_SCANNER_ID);
+				}
+
+				String reason = Constant.messages.getString("ascan.progress.label.skipped.reason.user");
+				getActiveScan(params).getHostProcesses().forEach(hp -> hp.pluginSkipped(pluginId, reason));
```

Comment:
I'm sure this is right but I'd like to understand this `->` syntax a bit better. I assume this works because `getHostProcesses()` returns an iterable. So `forEach` `hp` (host process) in the list (or whatever type of iterable) execute `pluginSkipped(pluginId, reason)`. Is that the correct interpretation?

Reply:
Yeah, that's it.